.github/workflows/client-targets.yml

```diff
@@ -34,6 +34,15 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Install Perl
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y perl
```

Comment:
Sorry for the silly question, but perl is definitely included in this runner: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2004-Readme.md

Is there something in the newer version of perl that we need?

Reply:
This is a remnant of me trying to get all the CI to run locally (with a different runner, since I couldn't get access to the exact gh one). Removing 👍